tests/core/test_save_restore.py

```diff
@@ -71,6 +71,17 @@ def __init__(self, cfg, trainer=None):
         else:
             self.temp_file = None
             self.temp_data = None
+        # mock child model
```

Comment:
These changes are separated to https://github.com/NVIDIA/NeMo/pull/5671 (nested NeMo models)